src/Umbraco.Tests.Integration/Umbraco.Tests.Integration.csproj

```diff
@@ -38,6 +48,9 @@
     </ItemGroup>
 
     <ItemGroup>
+      <None Update="Logging\UmbracoTraceLog.UNITTEST.20181112.json">
+        <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
Also, in the old project where you migrated those files from ([link](https://github.com/umbraco/Umbraco-CMS/blob/eed6c879780af555b2ce5b0306d0cdc2d37e4fb2/src/Umbraco.Tests/Umbraco.Tests.csproj#L534)) we had this as **Content** tag and the value of  `CopyToOutputDirectory` was `PreserveNewest`. Any reasons you changed that?

Reply:
No, that was a mistake